client/signup/config/steps.js

```diff
@@ -63,5 +63,18 @@ module.exports = {
 		},
 		dependencies: [ 'siteSlug' ],
 		providesDependencies: [ 'theme', 'images' ]
+	},
+
+	'mlb-themes': {
+		stepName: 'mlb-themes',
+		apiRequestFunction: stepActions.setThemeOnSite,
+		dependencies: [ 'siteSlug' ]
+	},
+
+	'mlb-domains': {
+		stepName: 'mlb-domains',
+		apiRequestFunction: stepActions.createSite,
+		providesDependencies: [ 'siteSlug' ],
+		delayApiRequestUntilComplete: true
```

Comment:
So, in this case the anonymous user should not be able to get back to domain screen?

I mean, If the user is logged, there's no extra step after selecting domain. But if the user is anonymous there's still one more screen (the `user` step) where there's a back button that would allow to alter the domain.